processor/attributesprocessor/README.md

```diff
@@ -56,6 +59,22 @@ For the `hash` action,
   action: hash
 ```
 
+
+For the `extract` action,
+ - `key` is required and will
```

Comment:
"and will", incomplete sentence?

Reply:
I actually can't remember what i was going to say there. I will just remove it.